vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,146 @@
+a static IP for a VM. See this [forum post](https://techcommunity.microsoft.com/t5/windows-insider-program/hyper-v-default-switch-ip-address-range-change-ver-1809-build/m-p/261431) for details. 
+ 
+Locally the problem is that the ENTANDO_DEFAULT_ROUTING_SUFFIX (in your entando.yaml or similar) is based on the 
+VM IP address. That setting is used to generate ingress routes so it's not a simple fix to
+rewire those settings once Entando is installed. In production settings there's generally a dedicated network layer to manage IPs/routing 
+(both on premise and cloud) but those options are generally not readily available in a local setup. Here are a 
+few options short of modifying your router/switch settings: 
+
+#### Option 1: Reinstall Entando
+Okay, this is a terrible option since you already installed it once and don't want to do it again every 
+time Windows surprises you with an update. Nevertheless, it works so it's worth mentioning.
+
+#### Option 2: Manually update your hosts file
+This is the simplest option but a bit annoying to do after every restart. 
+You need two pieces of information for this workaround and you'll need administrator access to do this.
+
+1. Determine the original IP used for your VM. This is included in the `ENTANDO_DEFAULT_ROUTING_SUFFIX` or you can 
+see it included in the ingress names. Run ` kubectl -n entando get ingress` and you should see something 
+like this:
+````
+NAME                          CLASS    HOSTS                                           
+quickstart-kc-ingress         <none>   quickstart-kc-entando.192.168.235.100.nip.io  
+quickstart-eci-ingress        <none>   quickstart-eci-entando.192.168.235.100.nip.io  
+quickstart-ingress            <none>   quickstart-entando.192.168.235.100.nip.io    
+````
+
+2. Determine the current IP using `hostname -I` in the VM or by running `multipass list` from Windows:
+```
+$ multipass list
+Name                    State             IPv4             Image
+primary                 Running           172.31.118.12   Ubuntu 18.04 LTS
+```
+
+3. As a Windows administrator, edit your hosts file (`C:\Windows\System32\drivers\etc\hosts`) to map any needed URLs from 
+the old IP to the new IP. This will bypass .nip.io lookups.
+
+```
+172.31.118.12 quickstart-entando.192.168.235.100.nip.io
+172.31.118.12 quickstart-kc-entando.192.168.235.100.nip.io
+172.31.118.12 quickstart-eci-entando.192.168.235.100.nip.io
+``` 
+
+4. You should now be able to access your Entando URLs via the new IP. If Entando was stalled starting up, it should
+continue starting up as soon as the external address is functional again. 
+
+#### Option 3: Add a Windows route.
+This is a little more involved but it means repairing your network settings can be done with a single command
+after the initial setup per VM. (Exercise for the reader, if you build a bat/cmd script to automate this repair, please
+share with the class!)
+
```

Comment:
Recommend removing this: (Exercise for the reader, if you build a bat/cmd script to automate this repair, please share with the class!)

I'd rather we direct them to a "bug/feature bounty" program where we pay them for the fix, or let them choose to share on their own. I think some people would do this naturally. If you decide that this is better for the perception of the company to leave it in, then I'd direct them to where they can contribute the script like the forums.